src/Tables/atoms/TableRowThumbnail.tsx

```diff
@@ -136,17 +149,44 @@ const TableRowThumbnail: React.FC<IProps> = ({ hoverZoom = true, image='', media
     timeoutRef.current = setTimeout(retryTimeout, randomDelay);
   },[retryCount, retryImageLoad, retryLimit, retryTimeout]);
 
-
-
   const onLoad = useCallback(()=>{
     timeoutRef.current && clearTimeout(timeoutRef.current);
     timeoutRef.current = null;
     setShowImage(true);
   },[]);
 
+  const checkIfImageExists = (url: string, callback: (exists: boolean) => void) => {
```

Comment:
callback is not a good name here. Use some different name like imageExistsCallback

Reply:
Done